.github/workflows/update-helm-index.yml

```diff
@@ -46,7 +46,14 @@ jobs:
           body: |
             Hi.
 
-            We released a new version of Inspektor Gadget, so the helm repository index needs to be updated.
+            We released a new version of Inspektor Gadget [${{ steps.bump_inspektor_gadget.outputs.version }}](https://github.com/inspektor-gadget/inspektor-gadget/releases/tag/${{ steps.bump_inspektor_gadget.outputs.version }}), so helm index needs to be updated.
+
+            This PR was automatically generated by a CI cron job:
+            * [update-helm-index.yml](https://github.com/inspektor-gadget/charts/blob/main/.github/workflows/update-helm-index.yml)
+
+            This PR will not be automatically merged. Please review, edit and merge as appropriate.
+            After merging, the following will be updated:
+            * [GitHub Page for Charts](https://inspektor-gadget.github.io/charts/index.yaml)
```

Comment:
Do you mean html instead of yaml?

Reply:
Yes, This PR updates the `index.yaml` so one can verify once the changes are merged. What do you think?

Reply:
Ah ok, that's fine then.
I merely wanted to have a way to discover the domain part of the URL.